### PR TITLE
PrSM output for top-down

### PR DESCRIPTION
**Pull request by zrolfs**
-Changed PSM output to PrSM
-Added unit test for whenever we get around to #2061

### CODE REVIEW

File: TaskLayer/SearchTask/PostSearchAnalysisTask.cs [hunk 1]
```diff
@@ -516,7 +516,10 @@ private void HistogramAnalysis()
 
         private void WritePsmResults()
         {
-            Status("Writing PSM results...", Parameters.SearchTaskId);
+            //if doing proteoform analysis, then output is proteoform-spectrum match (PrSM) instead of peptide-spectrum match (PSM)
+            string psmString = GlobalVariables.AnalyteType == GlobalVariables.Analyte.Proteoform ? "PrSM" : "PSM";
```
Comment (trishorts) [inline TaskLayer/SearchTask/PostSearchAnalysisTask.cs:520]: analyteString

File: Test/TopDownTest.cs [hunk 2]
```diff
@@ -47,13 +47,63 @@ public static void TestClassicSearchEngineTopDown()
...
-            new ClassicSearchEngine(allPsmsArray, listOfSortedms2Scans, variableModifications, fixedModifications, null, null, null, 
+            new ClassicSearchEngine(allPsmsArray, listOfSortedms2Scans, variableModifications, fixedModifications, null, null, null,
                 proteinList, searchMode, CommonParameters, null, null, new List<string>()).Run();
 
             var psm = allPsmsArray.Where(p => p != null).FirstOrDefault();
             Assert.That(psm.MatchedFragmentIons.Count == 47);
         }
 
+        /// <summary>
+        /// FIXME: MetaMorpheus currently reports ambiguity at the PrSM level, but starts tossing things when we get to the proteoform/protein level
```
Comment (trishorts) [inline Test/TopDownTest.cs:58]: Is this a "TODO"? We have used that terminology in the past. Maybe switch to that. And add an issue.

**Review by trishorts — APPROVED** [review]
This looks good to me. To very minor things to attend to ad libitum.

**Review by rmillikin — APPROVED** [review]

**Review by rmmiller22 — APPROVED** [review]

**Comment by rmmiller22** [comment]
Update the branch and then I will merge in the PR.

**Comment by rmmiller22** [comment]
This pull request is being reverted. Jenkins produced 0 PSMs once it (and another PR) were merged. You can re-open/ re-create this PR- and can be merged again once issues are identified and resolved.

**Pull request merged**